docs/en/examples/resume_training.md

```diff
@@ -1,3 +1,36 @@
 # Resume training
 
-Coming soon. Please refer to [chinese documentation](https://mmengine.readthedocs.io/zh_CN/latest/examples/resume_training.html).
+Resuming training means continuing training from the state saved from some previous training, where the state includes the model's weights, the state of the optimizer and the state of parameter scheduler.
+
+## Automatically resume training
+
+Users can set the `resume` parameter of [Runner](mmengine.runner.Runner) to enable automatic training resumption. If the latest checkpoint exists in `work_dir` (e.g. the training was interrupted during the last training), the training will be resumed from that checkpoint. If there is a latest checkpoint in `work_dir` (e.g. the training was interrupted during the last training), the training will be resumed from that checkpoint, otherwise (e.g. the last training did not have time to save the checkpoint or a new training task is started) the training will start again. Here is an example of how to enable automatic training resumption.
```

Comment:
```suggestion
Users can set the `resume` parameter of [Runner](mmengine.runner.Runner) to enable automatic training resumption. When `resume` is set to `True`, the Runner will try to load the latest checkpoint in `work_dir` automatically. If there is a latest checkpoint in `work_dir` (e.g. the training was interrupted during the last training), the training will be resumed from that checkpoint, otherwise (e.g. the last training did not have time to save the checkpoint or a new training task is started) the training will restart. Here is an example of how to enable automatic training resumption.
```